ibis/tests/all/conftest.py

```diff
@@ -201,6 +201,11 @@ def geo_df(geo):
     return None
 
 
+@pytest.fixture
```

Comment:
why are you doing it like this?

Reply:
as you asked me to use pandas backend to test the other ntile implementation I created that for the tests https://github.com/ibis-project/ibis/pull/2146/files#diff-0fbd582eeb40e92b4583db47d42143d3R216
let me know if you have another idea for that.